.vscode/sample.settings.json

```
@@ -1,14 +1,28 @@
 {
-    "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
-    },
-    "eslint.format.enable": true,
-    "eslint.workingDirectories": [
-        "./admin",
-        "./certif",
-        "./mon-pix",
-        "./api",
-        "./orga"
-    ],
-    "js/ts.implicitProjectConfig.experimentalDecorators": true
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "eslint.format.enable": true,
+  "eslint.workingDirectories": [
+    "./admin",
+    "./certif",
+    "./mon-pix",
+    "./api",
+    "./orga",
+    "./1d"
+  ],
+  "js/ts.implicitProjectConfig.experimentalDecorators": true,
+
+  // Only load the test files needed for the current test run (the default is to load all configured files)
+  "mochaExplorer.pruneFiles": true,
+  // The glob(s) describing the location of your test files (relative to the workspace folder)
+  "mochaExplorer.files": "api/tests/**/*_test.js",
+  // Shutdown the Mocha process (using process.exit()) after the last test has been run
+  "mochaExplorer.exit": true,
+  // The test-case timeout in milliseconds
+  "mochaExplorer.timeout": 20000,
+  // Use Mocha's experimental ESM module loader if it is available
+  "mochaExplorer.esmLoader": true,
```

Suggestion de config supplémentaire

```suggestion
  "mochaExplorer.esmLoader": true,
  "mochaExplorer.env": {
    "NODE_ENV": "test",
    // Avoids creating tests that only works in your timezone
    "TZ": "Asia/Kolkata",
     //"LOG_ENABLED": "true",
    //"LOG_FOR_HUMANS": "true",
    //"TEST_LOG_ENABLED": "true",
    // "LOG_LEVEL": "debug",
    // "DEBUG": "knex:*"
  },
```

---

Ahah, mon fameux kolkota! je rajoute pour documenter

---

Committed